packages/notebook/style/notebookfooter.css

```
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) Jupyter Development Team.
+ * Distributed under the terms of the Modified BSD License.
+ */
+
+.jp-Notebook-footer {
+  height: 10%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: var(--jp-cell-padding);
+  margin-left: calc(
+    var(--jp-cell-prompt-width) + var(--jp-cell-collapser-width) +
+      var(--jp-cell-padding)
+  );
+  border: var(--jp-border-width) solid var(--jp-cell-editor-border-color);
+  border-radius: 0;
+  color: var(--jp-layout-color3);
+}
+
+/* For devices that support hovering, hide checkboxes until hovered, selected...*/
+
+/* @media (hover: hover) {
+  .jp-Notebook-footer:hover {
+    border: var(--jp-border-width) solid var(--jp-cell-editor-border-color);
+    border-radius: 0;
+    color: var(--jp-layout-color3);
+  }
+}*/
+
+/* .jp-Notebook-footer:hover {
+  border: var(--jp-border-width) solid var(--jp-cell-editor-border-color);
+  border-radius: 0;
+  color: var(--jp-layout-color3);
+} */
+
+.jp-Notebook-footer.focused {
```

This should do the trick

```suggestion
.jp-Notebook-footer:focus {
```